readme.txt

```diff
@@ -3,6 +3,8 @@
 = [TBD] TBD =
 
 * Tweak - Rename the `Controller_Test_Case` `setUp` and `tearDown` methods and annotate them with `@before` and `@after` annotations to improve PHPUnit version cross-compat.
+* Fix - Add the dir and filename of `event-automator` in the Plugins_API so that the upsell CTA button text and links are shown properly in the Help section of TEC.
```

Comment:
For each new ticket always fork from `master` or the `release` branch, not from another `feature` branch (unless these are all tasks for a larger project in that case it would be a `bucket` branch).

Reply:
I continued from the previous PR because both issues were in the same file, `src/Tribe/Plugins_API.php`, and tbh it should have been one ticket, but ok yes you are right.